src/plugins/syscalls/win.h

```diff
@@ -3934,7 +3958,11 @@ SYSCALL(NtUserSetWindowDisplayAffinity, NTSTATUS, 0);
 SYSCALL(NtUserSetWindowFNID, NTSTATUS, 0);
 SYSCALL(NtUserSetWindowFeedbackSetting, NTSTATUS, 0);
 SYSCALL(NtUserSetWindowGroup, NTSTATUS, 0);
-SYSCALL(NtUserSetWindowLong, NTSTATUS, 0);
+SYSCALL(NtUserSetWindowLong, NTSTATUS, 4,
```

Comment:
Please move this above to be placed with the other defined syscalls

Reply:
Done.